lib/archive.ml

```diff
@@ -6,6 +6,20 @@
 
 open Bos_setup
 
+(* Safe wrapping for some Bytes functions *)
+
+module Sbytes = struct
```

Comment:
Might be worth moving it to a separate module, eventually a `Util` or `Stdext` where we can put all of these. That would encourage us and contributors to reuse the functions sitting there and eventually to contribute some new ones rather than inlining them elsewhere in the code!

Reply:
done